x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/use_preview_rule.ts

```diff
@@ -71,20 +71,25 @@ export const usePreviewRule = ({
     [advancedOptions]
   );
 
+  let isInvalidInterval = false;
   if (advancedOptions) {
     const timeframeDuration =
       (advancedOptions.timeframeEnd.valueOf() / 1000 -
         advancedOptions.timeframeStart.valueOf() / 1000) *
       1000;
 
     const { unit: intervalUnit, value: intervalValue } = getTimeTypeValue(advancedOptions.interval);
-    const { unit: lookbackUnit, value: lookbackValue } = getTimeTypeValue(advancedOptions.lookback);
     const duration = moment.duration(intervalValue, intervalUnit as 's' | 'm' | 'h');
-    duration.add(lookbackValue, lookbackUnit as 's' | 'm' | 'h');
     const ruleIntervalDuration = duration.asMilliseconds();
+    if (ruleIntervalDuration > timeframeDuration) {
+      isInvalidInterval = true;
+    }
 
     invocationCount = Math.max(Math.ceil(timeframeDuration / ruleIntervalDuration), 1);
     interval = advancedOptions.interval;
+
+    const { unit: lookbackUnit, value: lookbackValue } = getTimeTypeValue(advancedOptions.lookback);
+    duration.add(lookbackValue, lookbackUnit as 's' | 'm' | 'h');
```

Comment:
We don't include `lookbackValue` in invocation count invocation. Only the rule interval included. We use loopback value to calculate the `from` parameter. Which is the time where the first execution suppose to begin.

From the previous non-advanced version `from` was `now-6m` for one hour, `now-65m` for one day, `now-65m` for one week and `now-25h` for one month timeframes.

The only issue with the new `from` value I realised is that I should not calculate based on `now`, but rather on `timeframeEnd` as it can be flexible with the advanced options.

Reply:
In my original [PR](https://github.com/elastic/kibana/pull/137102) though the `lookbackValue` was involved in invocation count calculation, so this PR is fixing that issue #137660